packages/contracts/contracts/L1/L2OutputOracle.sol

```diff
@@ -26,14 +27,14 @@ contract L2OutputOracle is Initializable, Semver {
     uint256 public immutable L2_BLOCK_TIME;
 
     /**
-     * @notice The address of the challenger. Can be updated via upgrade.
+     * @notice The address of the colosseum contract. Can be updated via upgrade.
      */
-    address public immutable CHALLENGER;
+    address public immutable COLOSSEUM;
```

Comment:
why only `COLOSSEUM` contract address type is `address`, not `Colosseum` like below `VALIDATOR_POOL`?

Reply:
AFAIK, only the address of `Colosseum` contract is needed here.

Reply:
Thx! Then it's clear.